frontend/tests/features/filter/constants.ts

```diff
@@ -0,0 +1,240 @@
+import {
```

Comment:
Added this "pinned" test version of ontology term set as tests should be covering functionality not data; this prevents any future updates to schema from affecting tests.